db/migrate/20170525191310_add_unique_index_to_donations_transaction_id.rb

```diff
@@ -0,0 +1,5 @@
+class AddUniqueIndexToDonationsTransactionId < ActiveRecord::Migration
+  def change
+    add_index :donations, :transaction_id, unique: true
```

Comment:
You are going to get a little trouble if you try to run this on production... You'll have to fix a record (in production). 

Take a look in this 2 donations: 3903 and 3902